src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -118,14 +140,42 @@ export const pinTokensWithBalanceAtTop = ({
 };
 
 /**
- * Calculates total USD balance of the provided tokens list.
+ * Calculates total USD balance of the provided UI tokens list.
  *
- * @param tokens - The list of tokens for total USD balance calculation.
- * @returns The sum of tokens USD balance.
+ * @param tokens - The list of UI tokens for total USD balance calculation.
+ * @returns The sum of UI tokens USD balance.
  */
-export const sumTokensUsdBalance = (tokens: TokenUi[]): number =>
+export const sumTokensUiUsdBalance = (tokens: TokenUi[]): number =>
```

Comment:
if no particular reason, I would leave the name of this one as `sumTokensUsdBalance`

Reply:
or in any case, a separate PR if really needed

Reply:
Sure, I'll create a PR for that. The reason why I renamed it is to be able to create a method with the exact same purpose but for `Token` (not for `TokenUi`).

Reply:
The PR is ready - https://github.com/dfinity/oisy-wallet/pull/2258